app.py

```diff
@@ -12,12 +12,22 @@
 
 message_rate = int(Config.MESSAGE_RATE)  # Per second
 message_weightings = {
-    "RESPONDENT_AUTHENTICATED": 37,
-    "SURVEY_LAUNCHED": 35,
-    "RESPONSE_RECEIVED": 20,
-    "REFUSAL_RECEIVED": 5,
-    "FULFILMENT_REQUESTED": 2,
-    "UNDELIVERED_MAIL_REPORTED": 1
+    "RESPONDENT_AUTHENTICATED": 36,
+    "SURVEY_LAUNCHED": 34,
+    "RESPONSE_RECEIVED": 17,
+    "RESPONSE_RECEIVED.pqrs": 1,
+    "RESPONSE_RECEIVED.qm": 1,
+    "RESPONSE_RECEIVED.qm_blanks": 1,
+    "REFUSAL_RECEIVED": 1,
+    "FULFILMENT_REQUESTED.sms": 1,
+    "FULFILMENT_REQUESTED.print": 1,
+    "UNDELIVERED_MAIL_REPORTED": 1,
+    "FULFILMENT_CONFIRMED": 1,
+    "NEW_ADDRESS_REPORTED": 1,
+    "ADDRESS_NOT_VALID": 1,
+    "ADDRESS_TYPE_CHANGED": 1,
+    "ADDRESS_MODIFIED": 1,
+    "PRINT_CASE_SELECTED": 1
```

Comment:
Has any weighting been considered for these new types?  Or are these just holders.  e.g.  PRINT_CASE_SELECTED and RESPONSE_RECEIVED.pqrs are going to be quite large numbers, but in relatively short bursts - not evenly distributed.

Reply:
The card just said to set all the new events in this PR to 1 and for the total to sum to 100. It all works as it is right now so maybe something to change if/when it may cause issues?